README.md

```
@@ -382,21 +360,19 @@
 ## 🆘 Need Help?
 
 ### Container won't start?
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)